pkg/render/apiserver.go

```diff
@@ -41,8 +41,10 @@ const (
 
 var apiServiceHostname = apiServiceName + "." + APIServerNamespace + ".svc"
 
-func APIServer(installation *operator.Installation, aci *operatorv1beta1.AmazonCloudIntegration, tlsKeyPair *corev1.Secret, pullSecrets []*corev1.Secret, openshift bool) (Component, error) {
+func APIServer(installation *operator.Installation, aci *operatorv1beta1.AmazonCloudIntegration, tlsKeyPair *corev1.Secret, pullSecrets []*corev1.Secret, openshift bool, isManagement bool, tunnelCASecret *corev1.Secret) (Component, error) {
```

Comment:
Since the installation is being passed in I don't think we need the isManagement flag. (I'd be on board with dropping the openshift flag too)